.gitignore

```diff
@@ -50,6 +50,8 @@ local.properties
 # PDT-specific
 .buildpath
 
+# STS-specific
```

Comment:
can't see information about real test execution. Please check this. Expected something like this TestUnitTest > testname PASSED

Reply:
fixed